Tests/KernelTests/main.mm

```diff
@@ -542,12 +545,12 @@ - (void)testCachePerformance {
   std::for_each(times.begin(), times.end(), [&](const double d) {
     accum += (d - mean) * (d - mean);
   });
-  double stdev = sqrt(accum / (times.size()-1));
+  double stdev = sqrt(accum / (times.size() - 1));
 
-  if (mean > 1000 || stdev > 150) {
-    TFAILINFO("μ: %-3.2f σ: %-3.2f", mean, stdev);
+  if (mean > 80 || stdev > 1) {
```

Comment:
1 is a little low - especially when my machine is so much faster than yours 😝 
2 worked for me